spec/spec_helper.rb

```diff
@@ -53,13 +57,17 @@ def application_client
 end
 
 def user_client
-  Panoptes::Client.new(
+  @user_client ||= Panoptes::Client.new(
```

Comment:
I think that iinstead of memoizing here and resetting the client before each spec example run we can instead force the calling spec context to handle the client memoizing. 

It simplifies the spec setup / teardown (abstracted away in here) and clarifies the spec code imho. You are doing this in your specs anyway.

I may be missing something as to why this setup and teardown is necessary.

that would simplify the spec setup and avoid any issues

Reply:
you're probably right. i got stuck writing tests for a while because i realized that successive mentions of `user_client` were talking about two separate objects and so i got paranoid, but probably just defining a let binding will prevent that from being an issue

Reply:
Removed this behavior and all the tests still pass, as predicted.